src/components/Button/ButtonAddPostTopic.js

```diff
@@ -32,6 +35,12 @@ const ButtonAddPostTopic = ({topicName, onRefresh}) => {
   );
 };
 
+ButtonAddPostTopic.propTypes = {
+  topicName: PropTypes.string,
+  onRefresh: PropTypes.func,
+  followType: PropTypes.string
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on adding PropTypes for the new props. This will help with type checking and improve code reliability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->